Dockerfile

```
@@ -24,8 +24,13 @@ RUN set -eux \
     && cargo --version \
     && rustc --version \
     && apt-get remove -y --auto-remove wget \
-    && apt-get -y install gcc \
-    && rm -rf /var/lib/apt/lists/*
+    && apt-get -y install gcc
+
+# Make.
+RUN apt-get update && apt-get install -y make
+
+# Apt cleanup.
+RUN rm -rf /var/lib/apt/lists/*
```

```suggestion
    && apt-get -y install gcc make \
    && rm -rf /var/lib/apt/lists/*
```
these have to be a single `RUN` command, otherwise the cleanup won't have any effect since it would be in a different layer.